actors/builtin/miner/miner_actor.go

```diff
@@ -407,9 +406,8 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 		rt.Abortf(exitcode.ErrIllegalArgument, "seal challenge epoch %v too old, must be after %v", params.SealRandEpoch, challengeEarliest)
 	}
 
-	if params.Expiration <= rt.CurrEpoch() {
-		rt.Abortf(exitcode.ErrIllegalArgument, "sector expiration %v must be after now (%v)", params.Expiration, rt.CurrEpoch())
-	}
+	validateExpiration(rt, rt.CurrEpoch(), params.Expiration, params.SealProof)
```

Comment:
Small change to consolidate expiration validation, and to validate expiration as soon as possible.